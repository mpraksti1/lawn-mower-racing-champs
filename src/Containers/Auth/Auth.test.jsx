import React from 'react';
import { shallow } from 'enzyme';
import { Auth } from '.';

describe('Auth component', () => {
  it('renders without crashing', () => {
    shallow(<Auth loginFunc={() => {}} registerFunc={() => {}} isRequesting={false} />);
  });

  it('matches snapshot', () => {
    const wrapper = shallow(
      <Auth loginFunc={() => {}} registerFunc={() => {}} isRequesting={false} />,
    );

    expect(wrapper).toMatchSnapshot();
  });
});
