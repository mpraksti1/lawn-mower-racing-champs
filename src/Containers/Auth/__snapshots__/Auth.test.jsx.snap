// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Auth component matches snapshot 1`] = `
<div>
  <BGVideo
    dark={true}
    url="/assets/video/bgvid.mp4"
  />
  <withRouter(FormModal)
    isRequesting={false}
    login={[Function]}
    register={[Function]}
  />
</div>
`;
